.ruby-version.sample

```diff
@@ -0,0 +1 @@
+2.6.3@rubymem
```

Comment:
Can you remove the `@rubymem` part? we can't asume all use `rvm gemsets` and the usage of rvm.

Reply:
@bronzdoc Does it fail with rbenv or chruby?

Reply:
Yup.  e.g: 
```shell 
$ BUNDLE_GEMFILE=Gemfile.next bundle exec rspec 
rbenv: version `2.5.3@doc' is not installed (set by/Users/bronzdoc/projects/ombulabs/PackManager/.ruby-version)

Reply:
Got it. I’ll fix that. Thanks for checking!